docker/build/devdeps.manylinux.Dockerfile

```diff
@@ -50,7 +50,7 @@ RUN dnf install -y --nobest --setopt=install_weak_deps=False \
 ADD ./scripts/install_prerequisites.sh /scripts/install_prerequisites.sh
 ENV BLAS_INSTALL_PREFIX=/usr/local/blas
 ENV OPENSSL_INSTALL_PREFIX=/usr/local/openssl
-RUN dnf install -y --nobest --setopt=install_weak_deps=False \
+RUN dnf check-update && dnf install -y --nobest --setopt=install_weak_deps=False \
```

Comment:
This needs to be  changed back, otherwise the build will fail when the registry updates